x/auth/ante.go

```diff
@@ -85,7 +85,7 @@ func NewAnteHandler(am AccountMapper, fck FeeCollectionKeeper) sdk.AnteHandler {
 			signerAddr, sig := signerAddrs[i], sigs[i]
 
 			// check signature, return account with incremented nonce
-			signBytes := StdSignBytes(ctx.ChainID(), accNums[i], sequences[i], fee, msgs, stdTx.GetMemo())
+			signBytes := StdSignBytes(chainID, accNums[i], sequences[i], fee, msgs, stdTx.GetMemo())
```

Comment:
Does `ctx.ChainID()` not work? Why is this change in this PR?

Reply:
chainID comes from ctx.ChainID() - look at the few lines above - without this change signatures fail in lcd tests

Reply:
Probably somewhere we aren't passing `chain_id` correctly in the LCD request body then? The code on `develop` is confusing but it doesn't actually read from Viper.

Reply:
https://github.com/cosmos/cosmos-sdk/pull/1391